ethernet-view/src/components/ChartMenu/ChartList/ChartElement/ChartElement.tsx

```diff
@@ -0,0 +1,75 @@
+// @ts-ignore
+import CanvasJSReact from '@canvasjs/react-charts';
+import { ChartInfo, MeasurementInfo, Point } from "components/ChartMenu/types";
+import styles from "./ChartElement.module.scss";
+import { AiOutlineCloseCircle } from 'react-icons/ai'
+import { MutableRefObject, useCallback, useRef } from "react";
+
+var CanvasJSChart = CanvasJSReact.CanvasJSChart;
```

Comment:
It feels odd to define a variable to get the type of it later on, maybe you can try this later on the ref:
```typescript
useRef<typeof CanvasJSReact.CanvasJSChart>()
```